bin/require-qunit.js

```diff
@@ -0,0 +1,31 @@
+const resolve = require( "resolve" );
+
+// Depending on the exact usage, QUnit could be in one of several places, this
+// function handles finding it.
+module.exports = function requireQUnit() {
+	try {
+
+		// Something is borked here for tests
+
+		// First we attempt to find QUnit relative to the current working directory.
+		const localQUnitPath = resolve( "qunitjs", { basedir: process.cwd() } );
```

Comment:
I think you meant to use `resolve.sync(...)` instead of the async  `resolve(...)`